compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/focus/FocusTargetModifierNode.kt

```diff
@@ -36,4 +37,6 @@ sealed interface FocusTargetModifierNode : DelegatableNode {
  * This modifier node can be used to create a modifier that makes a component focusable.
  * Use a different instance of [FocusTargetModifierNode] for each focusable component.
  */
+
+@JsName("funFocusTargetModifierNode")
```

Comment:
JsName exists for commonMain on all platforms?

Reply:
yes

Reply:
Checked, it compiles on iOS as well